### ISSUE TITLE
Game Crash when playing a jukebox near a villager

### ISSUE THREAD

**Issue by curryducker** [jaacksondev/etched#44]
### What version are you seeing the problem on?

1.16.5

### Describe the issue

```markdown
1. grab a villager, jukebox and disc
2. play the disc near the villager
3. game crashes with a render error

It works fine when only having pollen and etched installed, but i can't find the incompatibility when having other mods and it would be great if the issue can be fixed so i can use the update in my modpack :)
Modlist is in latest.log
```

### latest.log

```shell
https://gist.githubusercontent.com/curryducker/0a0cb251b4bca1977a97111a662b2494/raw/38c47820a41e6064845f2b87ccebb2da7a68e572/latest.log

log was too long for this body so here's a link :)
```

**Comment by curryducker** [comment]
Incompatibility seems to be supplementaries, when i add it to the installation with only etched with pollen i have the same issue

**Comment by Ocelot5836** [comment]
Fixed in MoonflowerTeam/pollen@4ecb1804
